### PR TITLE
GitHub repo

### PR DESCRIPTION
**Pull request by LyzardKing**
Get the latest release versions with the github api.
Added to LightTable and Superpowers the api parses used for kotlin.
Fixed the mock server and the tests.

### CODE REVIEW

File: tests/medium/test_games.py [hunk 1]
```diff
@@ -73,16 +73,8 @@ class SuperpowersInContainer(ContainerTests, test_games.SuperpowersTests):
...
 
     def setUp(self):
-        self.hosts = {443: ["github.com"]}
+        self.hosts = {443: ["api.github.com", "github.com"]}
         self.apt_repo_override_path = os.path.join(self.APT_FAKE_REPO_PATH, 'superpowers')
         super().setUp()
         # override with container path
         self.installed_path = os.path.join(self.install_base_path, "games", "superpowers")
-
-    def test_install_with_changed_download_page(self):
```
Comment (didrocks) [inline tests/medium/test_games.py]: As you are overriding `get_metadata_and_check_license`, I still think it's important to override the download page in games and ides to fake the API returned values (seeding wrong content basically).

**Comment by didrocks** [comment]
Ok, just having this small comment (see in file), but other than that, it's good. Do not forget to uncomment the `return` on spring tools suite (both large/medium tests and the framework itself) so that you reenable the framework :)

WIth the change is taken into account and if the tests pass, +1 :) Thanks for looking at using the api, it seems way more robust to me!

**Comment by LyzardKing** [comment]
The spring-tools-suite hasn't changed.
It still needs fixing

**Comment by didrocks** [comment]
Sorry, I misread LightTable ;)

**Comment by didrocks** [comment]
+1 for those changes. Merging as well and many thanks :)

**Comment by LyzardKing** [comment]
Great!

**Pull request merged**